src/desktop/apps/gene/client.js

```diff
@@ -62,7 +63,11 @@ export const setupGenePage = () => {
       user={user ? user.toJSON() : null}
       mediator={mediator}
     >
-      <Contents {...options} onStateChange={onStateChange} />
+      <Theme>
```

Comment:
Might be better to wrap this in `<Boot>`?

Reply:
Ah yeah. Will do.